src/Parallel/centrality/pagerank.jl

```diff
@@ -1,27 +1,27 @@
 function pagerank(
     g::AbstractGraph{U},
-    α=0.85,
-    n=100::Integer,
-    ϵ=1.0e-6
-    ) where U <: Integer
+    α = 0.85,
+    n = 100::Integer,
+    ϵ = 1.0e-6,
+) where {U<:Integer}
```

Comment:
making this consistent with braces has been one of those goals that I've been putting off. Thanks for this.

Reply:
Oh, also. I think `=` in parameter lists should not have spaces. That is,
```
function foo(x::Integer; y=4, z="foo")
    a = 3
    b = "bar"
    c = Vector{Int}[1, 2, 3]
end
```